Vizzuality/half-earth-v3
# Fix/update nrc layers

## Update NRC landing layers to show updated country names on the tooltip
### Description
- Reuse SPI layer to show the boundaries and names on the NRC landing page
- Clean code from the deleted user config menu
- Clean some unused code from the release notes
- Text change
- 
### Testing instructions

- Go to NRC landing page, search or click on Czechia. You should see the correct name on the tooltip
- Also there the text change, (marine area)

### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-453?atlOrigin=eyJpIjoiNjM5Y2QxODBjNjVlNGExYzkwZWNiMWNmY2ZkNzZkNmQiLCJwIjoiaiJ9
